src/components/shared/buttons/OutlinedToggleButton.tsx

```diff
@@ -23,10 +23,14 @@ const OutlinedToggleButton = styled(ToggleButton)(({
         [`&.${toggleButtonClasses.selected}`]: {
             'backgroundColor': theme.palette[colorKey].alpha_12,
             'border': `1px solid ${theme.palette[colorKey].alpha_50}`,
-            'color': theme.palette[colorKey].dark,
+            'color':
+                theme.palette.mode === 'light'
+                    ? theme.palette[colorKey].dark
+                    : theme.palette[colorKey].main,
             '&:hover': {
                 border: `1px solid ${
-                    colorKey === 'success'
+                    theme.palette.mode === 'light' &&
+                    (colorKey === 'success' || colorKey === 'info')
```

Comment:
Fine with two - but I think these are usually best handles with `array` and then using `includes`. So we can easily add and remove items from the list.